pkgs/by-name/vp/vpp/package.nix

```diff
@@ -115,6 +116,10 @@ stdenv.mkDerivation rec {
 
   passthru.updateScript = nix-update-script { };
 
+  passthru.tests = {
+    inherit (nixosTests) vpp;
+  };
+
   meta = {
     description = "Fast, scalable layer 2-4 multi-platform network stack running in user space";
     homepage = "https://s3-docs.fd.io/vpp/${version}/";
```

Comment:
Github won't let me make a comment lower down, but one last `with` nitpick for the `meta.maintainers` on line 127